src/sagas/reown.js

```diff
@@ -778,13 +972,13 @@ export function* saga() {
     fork(init),
     takeLatest(types.SHOW_NANO_CONTRACT_SEND_TX_MODAL, onSendNanoContractTxRequest),
     takeLatest(types.SHOW_SIGN_MESSAGE_REQUEST_MODAL, onSignMessageRequest),
+    takeLatest(types.SHOW_SIGN_ORACLE_DATA_REQUEST_MODAL, onSignOracleDataRequest),
     takeLatest(types.SHOW_CREATE_TOKEN_REQUEST_MODAL, onCreateTokenRequest),
-    takeLeading('WC_SESSION_REQUEST', onSessionRequest),
```

Comment:
No need for this action handler anymore?

Reply:
Not there... We're processing `SESSION_REQUEST` messages in a saga that is forked, it's in line 361

It processes requests one at a time, making sure we don't lose messages.

This is part of the wallet connect reliability changes